person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(id, age, name: 'unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_service?
+    of_age? || @parent_permission
+  end
```

Comment:
[OPTIONAL] By default, all methods that are not decleared as private are usually public😉. Hence, I think you can just take the can_use_service? method to stay above the private method and there won't be any need declearing it as public again😉.

```suggestion
  def can_use_service?
    of_age? || @parent_permission
  end

  private

  def of_age?
    @age >= 18
  end
```